fleet_adapter_template/fleet_adapter_template/fleet_adapter.py

```diff
@@ -208,6 +208,31 @@ def make_callbacks(self):
             )
         )
 
+        callbacks.localize = lambda estimate, execution: self.localize(
+            estimate, execution
+        )
+
+        return callbacks
+
+    def localize(self, estimate, execution):
+        self.node.get_logger().info(
+            f'Commanding [{self.name}] to change map to'
+            f' [{estimate.map}]'
+        )
+        if self.api.localise(self.name, estimate.position, estimate.map):
```

Comment:
Use `localize`. Your callback is not consistent with your latest RobotClientAPI function